tests/auto/integration/tst_integration.cpp

```
@@ -147,6 +147,26 @@ class tst_integration : public QObject {
     pass.updateEnv();
   }
 
+  // Initialize a temporary pass store with an .gpg-id file and ImitatePass.
+  // Returns true on success, false on failure.
+  bool initImitateStore(QTemporaryDir &storeDir, ImitatePass &pass) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)